.github/workflows/extra_tests.yml

```diff
@@ -33,11 +33,18 @@ env:
 
 jobs:
   build_shadow:
+    env:
+      RUSTC_WRAPPER: sccache
+      RUSTV: stable
```

Comment:
Are these used anywhere?

Reply:
RUSTC_WRAPPER is used by cargo; added a comment.

RUSTV isn't, afaict. Removed.